### ISSUE TITLE
no docstrings for Zeros, Ones, Fill

### ISSUE THREAD

**Issue by CarloLucibello** [JuliaArrays/FillArrays.jl#95]
e.g.
```julia
help?> Ones
search: Ones ones leading_ones trailing_ones countlines count_ones to_indices RoundNearest RoundNearestTiesUp RoundNearestTiesAway count_zeros propertynames one oneunit isone exponent

  No documentation found.

  FillArrays.Ones is of type UnionAll.

  Summary
  ≡≡≡≡≡≡≡≡≡

  struct UnionAll <: Type{T}

  Fields
  ≡≡≡≡≡≡≡≡

  var  :: TypeVar
  body :: Any

  Supertype Hierarchy
  ≡≡≡≡≡≡≡≡≡≡≡≡≡≡≡≡≡≡≡≡≡

  UnionAll <: Type{T} <: Any
```

**Comment by dlfivefifty** [comment]
👍 A PR for this would be great.

**Comment by JeffFessler** [comment]
Probably this can be closed now due to #109
